client/gutenberg/extensions/markdown/editor.jsx

```diff
@@ -43,6 +43,8 @@ registerBlockType( 'a8c/markdown', {
 		</svg>
 	),
 
+	keywords: [ __( 'formatting' ), __( 'syntax' ), __( 'code' ) ],
```

Comment:
What do you think about adding _"markup"_ as well?

Reply:
Sure! But the limit is 3 keywords, so we should decide in favor of which one to do it. Perhaps in favor of `code`?

Reply:
Ah, I just noticed this limitation: _"It is only allowed to add as much as three terms per block."_  ([via](https://wordpress.org/gutenberg/handbook/block-api/#keywords-optional)).

Reply:
>Perhaps in favor of code

Yep, rather than searching with "code", there might be a higher chance of someone remembering "markdown" wrong and searching for it with "markup" instead.

Reply:
I agree, and TBH I was wondering between those two. Your feedback made it easy to decide 👍 

Done in https://github.com/Automattic/wp-calypso/pull/27266/commits/b671ca3828d4d5931f30eae1524f178cb5a54d38.